homeassistant/components/camera/__init__.py

```diff
@@ -275,6 +275,12 @@ def handle_async_mjpeg_stream(self, request):
 
         last_image = None
 
+        interval = .5  # default stream framerate: 2 fps
+        min_interval = .05  # max stream framerate: 20 fps
```

Comment:
I think this should be moved to a global constant.

Reply:
yes, you are right